docs/HowTo/Find-and-Connect/Specifying-NAT.md

```diff
@@ -0,0 +1,42 @@
+---
+description: Configuring NAT with Teku
+---
+
+# Configuring NAT
+
+Use the [`--p2p-nat-method`](../../Reference/CLI/CLI-Syntax.md#p2p-nat-method) option to specify the
+NAT method. Options are [`NONE`](#none) and [`UPNP`](#upnp).
+
+You cannot change the NAT method while Teku is running. To change the NAT method restart the node
+with the [`--p2p-nat-method`](../../Reference/CLI/CLI-Syntax.md#p2p-nat-method) option.
+
+## UPnP
+
+Specify `UPNP` to quickly allow inbound peer connections without manual router configuration. Use
+UPnP in home or small office environments where a wireless router or modem provides NAT isolation.
+
+UPnP automatically detects if a node is running in a UPnP environment and provides port forwarding.
+UPnP might introduce delays during node startup, especially on networks without a UPnP gateway
+device.
+
+!!! tip
+
+    UPnP support is often disabled by default in networking firmware. If disabled by default, you
+    must explicitly enable UPnP support.
+
+!!! important
+
+    When the NAT method is set to `UPNP`, the advertised port is the same as the
+    [listening port](../../Reference/CLI/CLI-Syntax.md#p2p-port).
```

Comment:
this statement is just completely incorrect, i'd remove this note.

The whole point of upnp, is you're telling the firewall to map you an external port to this service, and theres absolutely no guarantee that it'll be the same port you're using to offer the service, in fact it's probably fairly odd to have the same port used.

Reply:
Done.